cmd/tendermint/commands/inspect.go

```diff
@@ -0,0 +1,68 @@
+package commands
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+
+	cfg "github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/inspect"
+	"github.com/tendermint/tendermint/state"
+	"github.com/tendermint/tendermint/state/indexer/sink"
+	"github.com/tendermint/tendermint/store"
+	"github.com/tendermint/tendermint/types"
+)
+
+// InspectCmd is the command for starting an inspect server.
+var InspectCmd = &cobra.Command{
+	Use:   "inspect",
+	Short: "Run an inspect server",
+
+	RunE: runInspect,
+}
+
+func init() {
```

Comment:
is `--home` able to be used? What is the assumed path to the data directory?

Reply:
`--home` appears to be usable automatically. The `home` flag is applied in `setup.go` in the [PrepareBaseCmd](https://github.com/tendermint/tendermint/blob/630dda22346c171c0b8774b79216a862560c5657/libs/cli/setup.go#L30) function that is called on the `rootCmd` [in main.go](https://github.com/tendermint/tendermint/blob/630dda22346c171c0b8774b79216a862560c5657/cmd/tendermint/main.go#L50). This flag is then use to select the home directory in [cmd/setup.go](https://github.com/tendermint/tendermint/blob/630dda22346c171c0b8774b79216a862560c5657/libs/cli/setup.go#L135). The default value of this is `$HOME/.tendermint`, which is configured in [config/config.go](https://github.com/tendermint/tendermint/blob/630dda22346c171c0b8774b79216a862560c5657/config/config.go#L46)